### PR TITLE
Add Pico 4 controller models

### PR DESCRIPTION
**Pull request by rsjtdrjgfuzkfg**
This PR adds controller models for Pico 4 controllers, based on the controller models provided by Pico as part of their SDKs. Pico clarified to me that this is acceptable as long as the original license is attached and the provenance of the controller model is indicated. Furthermore, it adds rendered versions of the model for the WebXR interstitial widgets, and fixes various device type references to enable the correct interstitial to be shown.

*With this PR, I think all basic aspects for Pico 4 support are covered. Imho it would be great to get an ETA for official builds or, if there are still blockers, an official Pico support issue to track those.*

### CODE REVIEW

**Review by svillar — CHANGES_REQUESTED** [review]
OK I got the confirmation that everything is OK from the legal POV. Actually including the license might not be even needed, but it does not hurt.

Now to the point, 3D models are loaded properly but they have no texture applied. That's very likely because you have a single .mtl file for the models. Check the oculusvr or the hvr assets for example and you'd see how you need one for each controller. Also the name matters (they should have `_left` and `right` suffixes).

**Comment by rsjtdrjgfuzkfg** [comment]
I don't see why separating should change anything, but added a commit separating either way. Can you re-test?

If that does not work, feel free to add your own change that does work for you (replacing my separation commit, if necessary).

**Comment by svillar** [comment]
> I don't see why separating should change anything, but added a commit separating either way. Can you re-test?
> 
> If that does not work, feel free to add your own change that does work for you (replacing my separation commit, if necessary).

Indeed the change made no difference to me.

I've just realized what the problem is. I'm sorry for the confusion, I was testing your patch downloading it with `gh pr diff 369` because `gh pr checkout` failed due to conflicts. I don't know why, but the tex file was empty, so that's why textures were not applied.

Sorry for the back and forth, please do remove the extra commit, rebase against main and I'll merge it.

File: app/src/openxr/cpp/DeviceDelegateOpenXR.cpp [hunk 1]
```diff
@@ -99,6 +99,10 @@ struct DeviceDelegateOpenXR::State {
   bool mHandTrackingSupported = false;
 
   void Initialize() {
+#ifdef PICOXR
+    deviceType = device::PicoXR;
+#endif
```
Comment (svillar) [inline app/src/openxr/cpp/DeviceDelegateOpenXR.cpp:104]: This will be superseeded by PR #375 but we can leave it as it for now.

**Review by svillar — APPROVED** [review]
Many many thanks for your great contributions!

**Pull request merged**